LocalSettings.php

```diff
@@ -3436,6 +3436,12 @@
 		'default' => true,
 	],
 
+	// Page Images
+	'wgPageImagesNamespaces' = [
+		'default' => NS_MAIN,
+		'vgportdbwiki' => [NS_MAIN, NS_PLATFORM, NS_SERIES, NS_COMPANY],
```

Comment:
This should be reformatted so each constant is on a new line. Also, as for custom namespaces, I don't think the constants will work and need to be ID directly though not positive.

Reply:
I'm not even entirely sure what the constants are (if they even have one that is). Do you know where I'd be able to check?

Reply:
> I'm not even entirely sure what the constants are (if they even have one that is). Do you know where I'd be able to check?

If you see Special:ManageWiki/namespaces and select your namespace and select edit, the final subpage in the address bar is the namespace ID. (Custom ones are always above 3000)

Reply:
Oh, I know about the ID, but I wasn't sure about the constant. Also, id I were to try using the ID's, how would I type that in the code?

Reply:
Just replace the constants with the ID integer.